### PR TITLE
Privacy policy page

### PR DESCRIPTION
**Pull request by Jkrzy**
[Link to ZenHub issue.](https://app.zenhub.com/workspaces/doj-crt-intake-scrum-board-5d03bf56c1c8a35d482eca0f/issues/18f/crt-portal/314)

## What does this change?

Adds privacy policy page and removes temporary policy text in footer.

Building upon @cantsin's :awesome: work on `landing-page-layout`! Targeting that branch for this PR as a result.

Did my best to implement the mock but a few spacing areas were best guesses based on the visuals available. I also added `return to top` links for the 2nd and 3rd sections, this page is looooong 😃 

## (big) Screenshot (for front-end PR):
127 0 0 1_8000_privacy-policy(iPad Pro)
### Author

+ [x] If this is a story, run locally and check to make sure all Acceptance Criteria are met. (If any criteria are unclear, ask about them.)
+ [x] Check for [accessibility](/docs/a11y_plan.md).
+ [x] [Tests pass](https://github.com/USDOJ/crt-portal/#tests).

### Reviewer

+ [ ] If this is a story, run locally and check to make sure all Acceptance Criteria are met. (If any criteria are unclear, ask about them.)
+ [ ] [Tests pass](https://github.com/USDOJ/crt-portal/#tests).

## Notes for reviewer:

See [PR instructions doc](https://github.com/usdoj/crt-portal/blob/master/docs/pull_requests.md) for full pull request review instructions.

### CODE REVIEW

**Comment by LindsayYoung** [comment]
We need to add this page to the allow-list for pages that don't need auth: 
https://github.com/usdoj-crt/crt-portal/blob/7ccf721a74cc43b41a3530483dc956ad8a7a77b1/crt_portal/crt_portal/settings.py#L209-L213

**Comment by Jkrzy** [comment]
Tackled @LindsayYoung's comment in https://github.com/usdoj-crt/crt-portal/pull/491/commits/36b5211871559149b39effc702a7b9ea15bf934e

File: crt_portal/crt_portal/settings.py [hunk 1]
```diff
@@ -207,9 +207,11 @@
         "USERNAME_CLAIM": AUTH_USERNAME_CLAIM,
         "GROUP_CLAIM": AUTH_GROUP_CLAIM,
         'LOGIN_EXEMPT_URLS': [
-            '',
-            'report/',
-            'robots.txt',
+            '^$',
```
Comment (LindsayYoung) [inline crt_portal/crt_portal/settings.py:210]: Thank you for the fix on this!!

File: crt_portal/cts_forms/templates/base.html [hunk 2]
```diff
@@ -54,10 +54,6 @@
       {% include "partials/footer.html" %}
       </div>
```
Comment (LindsayYoung) [inline crt_portal/cts_forms/templates/base.html:56]: I think this needs to move to the report_base

File: crt_portal/cts_forms/templates/partials/policy.html [hunk 3]
```diff
@@ -1,30 +0,0 @@
-{% load i18n %}
```
Comment (LindsayYoung) [inline crt_portal/cts_forms/templates/partials/policy.html:1]: We still need this on the form pages- just confirmed with Dan

File: crt_portal/cts_forms/templates/partials/policy.html [hunk 4]
```diff
@@ -1,30 +0,0 @@
...
-    <div class="grid-col-12">
-      <div class="usa-footer__heading usa-footer__logo-heading">
-        Privacy Policy
-        <div class="usa-footer__separator"></div>
-      </div>
-
-      <div class="grid-row grid-gap">
-        <div class="grid-col-9 usa-footer__links" style="font-size: 14px">
-          <p style="margin-bottom: 1em">
-            The purpose of this form is to allow the public to submit civil rights complaints to the Department of Justice, thereby allowing us to enforce over thirty civil rights statutes within our authority. <a style="color: white; text-transform: none" href="https://www.justice.gov/crt/privacy-policy">These statutes</a> authorize us to collect this information.  You should know that any information you provide through this form is voluntary, yet failure to provide some of the information might limit the Department’s ability to pursue your claim. We may use this information for certain routine uses, including sharing this information under certain circumstances with:
```
Comment (LindsayYoung) [inline crt_portal/cts_forms/templates/partials/policy.html:15]: Update this link to the new privacy policy

File: package.json [hunk 5]
```diff
@@ -26,7 +26,9 @@
...
...onfig pa11y_scripts/.p1-error.json && echo 2 && pa11y-ci --config pa11y_scripts/.p2-primary-issue.json && echo 2-error && pa11y-ci --config pa11y_scripts/.p2-error.json && echo 3-hatecrime && pa11y-ci --config pa11y_scripts/.p3-hatecrime.json && echo 3-commercial && pa11y-ci --config pa11y_scripts/.p3-location-commercial.json && echo 3-education && pa11y-ci --config pa11y_scripts/.p3-location-education.json && echo 3-police && pa11y-ci --config pa11y_scripts/.p3-location-police.json && echo 3-workplace && pa11y-ci --config pa11y_scripts/.p3-location-workplace.json && echo 4 && pa11y-ci --config pa11y_scripts/.p4-protected-class.json && echo 5 && pa11y-ci --config pa11y_scripts/.p5-date.json && echo 5-error && pa11y-ci --config pa11y_scripts/.p5-errors.json && echo 6 && pa11y-ci --config pa11y_scripts/.p6-details.json && echo 7 && pa11y-ci --config pa11y_scripts/.p7-review && echo 8 && pa11y-ci --config pa11y_scripts/.p8-confirmation && echo view && pa11y-ci --config pa11y_scripts/.crt-view.json && echo details && pa11y-ci --config pa11y_scripts/.crt-details.json && echo new && pa11y-ci --config pa11y_scripts/.crt-new.json && pa11y-ci --config pa11y_scripts/.crt-privacy-policy.json"
```
Comment (LindsayYoung) [inline package.json:31]: 🥇

File: crt_portal/cts_forms/templates/partials/policy.html (hunk 3 above)
Comment (Jkrzy) [reply crt_portal/cts_forms/templates/partials/policy.html]: Ahh good call, that makes alot of sense! Re-added.

File: crt_portal/cts_forms/templates/partials/policy.html (hunk 4 above)
Comment (Jkrzy) [reply crt_portal/cts_forms/templates/partials/policy.html:15]: ✔️

**Comment by Jkrzy** [comment]
Thanks @LindsayYoung, requested changes are in!

**Comment by cantsin** [comment]
Ooh. If we are going to keep the older privacy footer then there are some things I want to clean up (mostly moving out all the `style="..."`)

**Comment by Jkrzy** [comment]
Rebased against develop

**Comment by Jkrzy** [comment]
Merging on @cantsin's 👍 review

**Pull request merged**